.github/workflows/create_deploy_pr.yml

```diff
@@ -0,0 +1,58 @@
+name: Create Deploy Workflow PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      repo:
+        description: The Repository the PR will be created in
+        required: true
+      file_name:
+        description: The file to include in the PR
+        required: true
+        default: deploy-template.yml
+
+jobs:
+  create_pull_request:
+    name: Create Branch & PR
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-gov-west-1
+
+      - uses: marvinpinto/action-inject-ssm-secrets@v1.1.1
+        with:
+          ssm_parameter: /dsva-vagov/platform-console-api/utility/github_access_token
+          env_variable_name: GITHUB_ACCESS_TOKEN
+
+      - uses: actions/checkout@v2
+
+      - name: Copy File
+        run: cp .github/workflows/${{env.INPUT_FILE_NAME}} ${{runner.temp}}
```

Comment:
Should this (and the other references) be github.event.inputs.file_name instead of env.INPUT_FILE_NAME?

Reply:
It will also create an environment variable: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs

> When you specify an input in a workflow file or use a default input value, GitHub creates an environment variable for the input with the name INPUT_<VARIABLE_NAME>. The environment variable created converts input names to uppercase letters and replaces spaces with _ characters.

Reply:
Oh I didn't know this TIL! Awesome